--- a/src/gui/sdlmain.cpp	2018-04-01 17:06:48.726777671 -1000
+++ b/src/gui/sdlmain.cpp	2018-04-01 17:07:10.407232399 -1000
@@ -1715,7 +1715,7 @@
 	Pstring = sdl_sec->Add_path("mapperfile",Property::Changeable::Always,MAPPERFILE);
 	Pstring->Set_help("File used to load/save the key/event mappings from. Resetmapper only works with the default value.");
 
-	Pbool = sdl_sec->Add_bool("usescancodes",Property::Changeable::Always,true);
+	Pbool = sdl_sec->Add_bool("usescancodes",Property::Changeable::Always,false);
 	Pbool->Set_help("Avoid usage of symkeys, might not work on all operating systems.");
 }
 
